### PR TITLE
added a credit deposit to pubbystation's vault

### LINKED ISSUE #21766: No credit deposit on pubby
No credit deposit. Found out when I had >5000 credits in coins and bank marks

### PR DESCRIPTION
**Pull request by Mervill**
:cl: Mervill
add: Added a credit deposit to pubbystation's vault
del: Removed a rather garish golden statue of the HoP from pubbystation's vault
/:cl:

fixes #21766

### CODE REVIEW

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 1]
```diff
@@ -9906,8 +9901,13 @@
...
 /area/bridge)
 "aue" = (
+/obj/structure/closet/secure_closet/freezer/money,
+/obj/item/weapon/reagent_containers/food/drinks/bottle/vodka/badminka,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/clothing/head/bearpelt,
+/obj/item/weapon/skub,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
-/obj/structure/statue/gold/hop,
```
Comment (Iamgoofball) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: :-1:

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 2]
```diff
@@ -9906,8 +9901,13 @@
 /turf/open/floor/plating,
 /area/bridge)
 "aue" = (
+/obj/structure/closet/secure_closet/freezer/money,
+/obj/item/weapon/reagent_containers/food/drinks/bottle/vodka/badminka,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/clothing/head/bearpelt,
+/obj/item/weapon/skub,
```
Comment (Iamgoofball) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: :+1:

Comment (Mervill) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: This was just moved from the top-left of the vault to the lower left (where the HoP statue was)

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 1 above)
Comment (Mervill) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: Not really anywhere good to put this, all the wall space is taken up, it was either this or the skub

**Pull request merged**